torchmetrics/text/squadv2/evaluate_squad.py

```diff
@@ -0,0 +1,310 @@
+import argparse
```

Comment:
If this script is copied from somewhere else then we need to include a reference.
Also can we remove anything non-essential and rename to something else like `helper.py`?

Reply:
moreover, can you explain why such a script is even needed?

Reply:
It's the helper script that has all the required functions for running the metric